### PR TITLE
Fixing RTL alignment for navbar (FND-1489)

### PR DESCRIPTION
**Pull request by aaka3207**
PR for FND-1489 to properly align navbar elements for RTL languages (Including studio).

1. Menu drop-down padding
**Before**
image

**After**
image

2. Create menu option alignment (Only applies to Studio)
**Before**
image

**After**
image

3. User menu option alignment

**Before**
image

**After**
image

4. Help menu option alignment

**Before**
image

**After**
image

5. Hamburger menu options

**Before**
image

**After**
image

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

**Comment by aaka3207** [comment]
tagging @dju90

**Comment by dju90** [comment]
Nit: the padding between the "down triangle" icon and the button text in the Create and User menus (first two screenshots) seems a little narrower compared to Left-to-Right mode. In the inspector in LTR mode, I see a "&nbsp" (space) between the label text and the triangle, which has a width of 12.61 pixels. Is that still present in the RTL version?
image

**Comment by aaka3207** [comment]
> Nit: the padding between the "down triangle" icon and the button text in the Create and User menus (first two screenshots) seems a little narrower compared to Left-to-Right mode. In the inspector in LTR mode, I see a "&nbsp" (space) between the label text and the triangle, which has a width of 12.61 pixels. Is that still present in the RTL version?
> image

@dju90 that &nbsp is still there, that seems to be a calculated value. It's likely because the text is in a `<span>` tag, which usually has better spacing in LTR format, but since the RTL fix doesn't change the HTML here (just the css) that extra space is being removed. 

I can add a little extra padding, just need to make sure it looks right on mobile too.

File: shared/css/user-menu.scss [hunk 1]
```diff
@@ -131,7 +145,7 @@
     .hide-mobile {
       display: none;
     }
-                                                                                              }
+  }
```
Comment (daynew) [inline shared/css/user-menu.scss:148]: 😅

**Review by daynew — APPROVED** [review]
Looks good! Just need to wait for the `staging` branch to be re-opened so we can merge this in.

**Pull request merged**